### DISCUSSION TITLE
Enabling full-width in one portion of a blog page

### DISCUSSION THREAD

**Discussion by jmlipman** [quarto-dev/quarto-cli#4970] [Q&A] [answered]
Hi,

Is it possible to enable full-width to only one specific portion of a blog entry? I would like to achieve something like this: https://ivelasq.quarto.pub/building-a-blog-with-quarto/lets-build-a-quarto-blog/#type

Thanks!

**Comment by mcanouil** [comment, marked as answer]
Yes, use a fenced div with appropriate class, see <https://quarto.org/docs/authoring/article-layout.html#overflowing-content>.

Note that the source code of the page you mentioned is included via a GitHub icon on the top right.
See the part of the code you are interested to replicate: <https://github.com/ivelasq/2022-08-30_building-a-blog-with-quarto/blob/main/lets-build-a-quarto-blog/index.qmd#L185-L208>.
